timesketch/frontend-ng/src/components/Explore/AggregateDialog.vue

```diff
@@ -0,0 +1,537 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-card class="mx-auto">
+    <v-card-title>
+      <span class="headline">Event Data Analytics</span>
+      <v-spacer></v-spacer>
+      <v-btn icon @click="clearAndCancel">
+        <v-icon>mdi-close</v-icon>
+      </v-btn>
+    </v-card-title>
+    <v-card-subtitle>
+      <span class="text-h6">Field:&nbsp;<span style="font-family: monospace">{{ eventKey }}</span> | Value: &nbsp;<span style="font-family: monospace">{{ eventValue }}</span></span>
+    </v-card-subtitle>
+    <v-card-text>
+      <v-container fluid>
+        <v-row justify="center">
+          <v-col>
+            <v-card height="450px" :loading="!statsReady">
```

Comment:
We don't use the elevation (shadow) on cards in the UI. add the `outlined` attribute to switch to the flat version.

Reply:
Done